config/locales/en/shared.yml

```diff
@@ -322,8 +322,19 @@ en:
         employment: Employment income
         employment_question: Do you need to tell us anything else about your client's employment?
         employment_details: Details
-      income_cash_payments:
-        heading: Payments your client receives in cash
+      benefits_income:
+        applicant:
+          heading: Clients benefits
```

Comment:
Should there be an apostrophe in the headings - Client's and Partner's?

Reply:
No, but then I shouldn't have pluralised the descriptions! 
It should have been `Client benefits` and `Partner benefits` I have updated now :+1: